orttraining/orttraining/models/runner/pipeline.h

```diff
@@ -141,6 +147,18 @@ class PipelineScheduler {
   size_t GetScheduleSize() const { return compute_commute_table_.size(); }
   // Number of stages.
   size_t GetStageSize() const { return num_stages_; }
+  std::vector<PipelineSlot> GetSchedule(const int stage_id) const {
```

Comment:
>GetSchedule [](start = 28, length = 11)

This only returns commute schedule and leaves out compute schedule. Maybe rename it to `GetCommuteSchedule()` to make it more precise?